infrastructure/tests/test_data_import.py

```diff
@@ -124,3 +126,63 @@ def test_year_error(self):
         self.assertEquals(AnnualSpendFile.objects.all().count(), 1)
         spend_file = AnnualSpendFile.objects.first()
         self.assertEquals(spend_file.status, AnnualSpendFile.ERROR)
+
+
+    def test_upload_project(self):
+        """Scope of Test: With no existing projects run an upload and check that the correct fields are populated"""
+        self.assertEquals(BudgetPhase.objects.all().count(), 5)
+
+        with open('infrastructure/tests/test_files/test.xlsx', 'rb', ) as f:
```

Comment:
let's test this at a lower level to close in on the unit under test.
that also allows us to use a data structure where we can more easily see the data used in the test, and when it changes, what the change was. While excel doesn't offer diffing like plain text does.

Reply:
Added mock data and using the lower level load_file instead of load_excel